mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -220,8 +227,22 @@ class LinearSVC @Since("2.2.0") (
         None
       }
 
-      val getAggregatorFunc = new HingeAggregator(bcFeaturesStd, $(fitIntercept))(_)
-      val costFun = new RDDLossFunction(instances, getAggregatorFunc, regularization,
+      val standardized = instances.map {
```

Comment:
Move the standardization outside of `HingeAggregator`, so that no longer need to standardize input in each iter.